src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,9 +8,8 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "a/123, Flower Street, #05-239, 766543";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must be in this format: a/BLOCK, STREET, UNIT, POSTAL CODE";
 
     public final String value;
```

Comment:
No need to maintain 'value' anymore. 
Use class variables of type Block, Street, Postalcode , etc for this purpose.